miriway.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Miriway
 Comment=Mir/Wayland desktop environment
-Exec=miriway-session
+Exec=sh -lc miriway-session
```

Comment:
Wouldn't adding `-l` to the shebang line have worked instead?

Reply:
That gave a crazy error I wasn't motivated to track down